runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -737,13 +758,20 @@ func s:EndTermDebug(job_id, exit_code, event)
     doauto <nomodeline> User TermdebugStopPre
   endif
 
+  if s:asmbuf > 0
+    exe 'bwipe! ' . s:asmbuf
+  endif
+  if s:varbuf > 0
+    exe 'bwipe! ' . s:varbuf
+  endif
```

Comment:
This stuff should be in EndDebugCommon too, otherwise this doesn't apply to prompt mode